src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/SimpleConfigModel.cs

```diff
@@ -517,6 +606,32 @@ public static SimpleConfigModel FromJObject(JObject source)
             config.Tags = source.ToStringDictionary(StringComparer.OrdinalIgnoreCase, nameof(config.Tags));
             config.PostActionModel = RunnableProjects.PostActionModel.ListFromJArray((JArray)source["PostActions"]);
 
+            // Custom operations at the global level
+            JToken globalCustomConfigData = source[nameof(config.CustomOperations)];
+            if (globalCustomConfigData != null)
+            {
+                config.CustomOperations = CustomFileGlobModel.FromJObject((JObject)globalCustomConfigData, string.Empty);
+            }
+            else
+            {
+                config.CustomOperations = null;
+            }
+
+            // Custom operations for specials
+            IReadOnlyDictionary<string, JToken> allSpecialOpsConfig = source.ToJTokenDictionary(StringComparer.OrdinalIgnoreCase, "SpecialCustomOperations");
```

Comment:
I've also been trying to come up with a better name for these things. I've started to use "FileGlob" as part of the names of some related symbols. So for specials, the best rename I've come up with is FileGlobSpecificOperations or just FileGlobOperations. I'm not 100% satisfied with that name, so haven't made changes yet.